server.py

```
@@ -55,23 +55,335 @@
+        # before _main wires routes) — harmless.
+        pass
+
+    if not is_setup_complete():
+        _graph = None
+        log.info("[reload] setup not complete — config reloaded, graph not compiled")
+        return True, "config reloaded • setup not complete"
+
+    try:
+        new_graph = create_agent_graph(new_config)
+    except Exception as e:
+        log.exception("[reload] graph rebuild failed: %s", e)
```

Ruff (0.15.10)

[warning] Redundant exception object included in `logging.exception` call

(TRY401)